### ISSUE TITLE
Design | Profile | Update Sketch files: Square the rounded corners in Profile

### ISSUE THREAD

**Issue by fmccaf1** [department-of-veterans-affairs/va.gov-team#55723]
## Background

@adamwhitlock1 let us know that rounded corners in boxes within Profile require custom code, as this is not a pattern throughout the VA.gov site. Instead of keeping and maintaining the custom code, we will move forward with squaring the rounded corners and remove this custom code in Profile.

## Tasks

- [ ] Review Sketch files and square content box corners that are currently rounded 
  - [x] Account security
  - [x] Contact information
  - [x] Direct Deposit
  - [x] Personal information
  - [x] Military information
  - [x] Connected Apps

## Acceptance criteria

- [x] All Sketch files have been updated accordingly

**Comment by mtcA6** [comment]
@pamela-drouin @andaleliz could you kindly point this ticket porfa?

**Comment by andaleliz** [comment]
@adamwhitlock1 has the code already been removed from the FE or are you waiting for the Sketch files to be updated first?

**Comment by adamwhitlock1** [comment]
corners have been squared / code removed from the FE, so I'm not waiting on any Sketch files or anything like that @andaleliz. If you see anywhere that they are still rounded, let me know.

**Comment by adamwhitlock1** [comment]
Secondary / somewhat unrelated question. Why does Notification Settings not use the card/table style component at all?

**Comment by andaleliz** [comment]
Thanks @adamwhitlock1 and great question! @mtcA6 asked me that recently too so the answer is fresh in my mind.

We first explored use those tables along with a read/edit pattern like the rest of profile ([prototype](https://preview.uxpin.com/5ecec4de6ec34eddc9f095677a881be3646124ce?_gl=1*1mmxh3z*_gcl_au*NTU2MTY1OTA5LjE2ODUxMjg3MDY.#/pages/139049740/simulate/no-panels?mode=i)).  However, user feedback + the tech constraints led us away from the pattern in the rest of profile.

I honestly can't remember with 100% certainty why we took the notifications out of cards.  But, I'm ~75% sure it was for the following reasons:
- We know the page had to be scalable and accommodate notification groups with only one item, and one channel.  It felt like overkill to put it in a table
- The tables have a lot of padding and margin, and used up real estate for no real good reason we could think of, especially since there were no buttons to visually contain and relate to data or inputs
- If it followed a different interaction pattern, it could follow a different visual pattern too. We wanted to try something new

I wish we had documented it better but that's the best I can remember for the moment.  

As an aside: Travis also asked how we would know if we should use the table pattern on the page - feedback from Veterans is what came to my mind. I'm not sure how else we'd know for sure if we should change it. I'm totally open to it if it works better (or at least the same) for people.
